scripts/run_synth_ipbb.py

```diff
@@ -19,13 +19,13 @@
     'mp7xe_690': 'xe',
 }
 
-DefaultVivadoVersion = os.getenv('VIVADO_VERSION')
+DefaultVivadoVersion = os.getenv('UGT_VIVADO_VERSION')
 if not DefaultVivadoVersion:
-    raise RuntimeError('VIVADO_VERSION is not defined.')
+    raise RuntimeError('UGT_VIVADO_VERSION is not defined.')
 
-VivadoBaseDir = os.getenv('VIVADO_BASE_DIR')
+VivadoBaseDir = os.getenv('UGT_VIVADO_BASE_DIR')
 if not VivadoBaseDir:
-    raise RuntimeError('VIVADO_BASE_DIR is not defined.')
+    raise RuntimeError('UGT_VIVADO_BASE_DIR is not defined.')
 
 vivadoPath = os.path.abspath(os.path.join(VivadoBaseDir, DefaultVivadoVersion))
 if not os.path.isdir(vivadoPath):
```

Comment:
Duplicate, the vivado path is validated in function `main()`, as the version defined in environ can be overwritten by command line flag `--vivado`).